packages/payment-experience-api/src/api/paytrailCardRedirectSuccessController.spec.ts

```diff
@@ -0,0 +1,447 @@
+/* eslint-disable */
+
+import { PaytrailCardRedirectSuccessController } from './paytrailCardRedirectSuccessController'
+import type { Request, Response } from 'express'
+import axios from "axios"
+import { PaymentType } from "@verkkokauppa/payment-backend"
+
+jest.mock('@verkkokauppa/payment-backend')
+jest.mock('@verkkokauppa/product-backend')
+jest.mock('@verkkokauppa/configuration-backend')
+jest.mock('@verkkokauppa/order-backend', () => {
+ // grab all the *real* implementations of the module's functions
+  // in an object
+  const actual = jest.requireActual('@verkkokauppa/order-backend')
+  // return a new module implementation
+  return {
+    __esModules: true,
+    // first start with all of the module's functions auto-mocked
+    getOrderAdmin: jest.fn(() => []),
+    // lastly override w/ any of the module's functions that
+    // we want to use the *real* implementations for
```

Comment:
Add comments why this was made, because axiosMock returns are mocked in function

Reply:
added comment, rows 23 and 24